js/templates/modals/settings/moderation.html

```diff
@@ -29,23 +32,133 @@ <h2 class="h3 clrT">Moderation</h2>
                 <label for="settingsModerationStatusFalse">Off</label>
               </div>
             </div>
           </div>
         </div>
         <div class="flexRow">
           <div class="col3">
-            <label for="settingsModeratorDescription" class="required">Decription</label>
+            <label class="required" for="moderationFeeType"><%= ob.polyT('settings.moderationTab.feeTypeAmount') %></label>
           </div>
           <div class="col9">
-            <% if (ob.errors.description) print(ob.formErrorTmpl({ errors: ob.errors.description })) %>
-            <textarea rows="3" maxlength="160" name="modInfo.description" id="settingsModeratorDescription"
-                      class="clrBr clrSh2" placeholder="Describe yourself"><%= ob.description %></textarea>
+            <% if (ob.errors['modInfo.fee.feeType']) print(ob.formErrorTmpl({ errors: ob.errors['modInfo.fee.feeType'] })) %>
+            <% if (ob.errors['modInfo.fee.fixedFee.feeType']) print(ob.formErrorTmpl({ errors: ob.errors['modInfo.fee.fixedFee.feeType'] })) %>
+            <div class="flexRow gutterH rowTn">
+              <div class="col4">
+                <select id="moderationFeeType" name="modInfo.fee.feeType" class="clrBr clrP clrSh2">
+                  <option value="PERCENTAGE" <% if (!ob.fee || (ob.fee && ob.fee.feeType === 'PERCENTAGE')) print('selected') %>>
+                    <%= ob.polyT('settings.moderationTab.percentage') %>
+                  </option>
+                  <option value="FIXED" <% if (ob.fee && ob.fee.feeType === 'FIXED') print('selected') %>>
+                    <%= ob.polyT('settings.moderationTab.fixed') %>
+                  </option>
+                  <option value="FIXED_PLUS_PERCENTAGE" <% if (ob.fee && ob.fee.feeType === 'FIXED_PLUS_PERCENTAGE') print('selected') %>>
+                    <%= ob.polyT('settings.moderationTab.fixedPlusPercentage') %>
+                  </option>
+                </select>
+              </div>
+              <div class="col2 js-feePercentageInput <% if (ob.fee && ob.fee.feeType === 'FIXED') { print('visuallyHidden') } %>">
+                <div class="inputPercentWrapper clrBr clrSh2">
+                  <% var perc = ob.fee.percentage || 0; %>
+                  <input type="text"
+                         maxlength="5"
+                         name="modInfo.fee.percentage"
+                         value="<%= perc %>"
+                         data-var-type="number">
+                </div>
+              </div>
+              <div class="col2 js-feeFixedInput <% if (!ob.fee || (ob.fee && ob.fee.feeType === 'PERCENTAGE')) { print('visuallyHidden') } %>">
+                <% var amt = ob.fee && ob.fee.fixedFee ? ob.fee.fixedFee.amount : 0;
+                   var ccode = ob.fee && ob.fee.fixedFee ? ob.fee.fixedFee.currencyCode : ob.defaultCurrency;
+                   amt = typeof amt === 'number' ? Number(ob.formatPrice(amt, ccode === 'BTC')) : amt;
```

Comment:
This was here to truncate trailing zeros when formatPrice returned values like 0.00 or 0.00000000. After merging with master and retesting, the new formatPrice code prevents that issue, so I'll take this out.